src/FsAutoComplete.Core/UnusedDeclarationsAnalyzer.fs

```diff
@@ -0,0 +1,48 @@
+//Based on VF# implementation - https://github.com/Microsoft/visualfsharp/blob/master/vsintegration/src/FSharp.Editor/Diagnostics/UnusedDeclarationsAnalyzer.fs
```

Comment:
I'd prefer to use paket and directly download + use that file for `microsoft/visualfsharp`

If code need to be changed (to remove things), maybe can be done with minimal `#if def` in vf# repo.
So upgrade is less painful later, and less code to maintain too

Reply:
It's not exactly the file from VF# as it contains VS specific things.

Reply:
Yes, that's why i am proposing to send a PR to vf# repo, and add #ifdef , to stop VS specific things
so we can paket pull these files directly